app/src/views/ActiveSurgeDeployments/OngoingERUDeployments/index.tsx

```diff
@@ -1,9 +1,404 @@
+                    ? new Date(eru.end_date).getTime()
+                    : undefined),
+            ) ?? [],
+        ).filter(isDefined);
+
+        const start = minSafe(startDateList);
+        const end = maxSafe(endDateList);
+
+        if (isNotDefined(start) || isNotDefined(end)) {
+            return { start: new Date(), end: new Date() };
+        }
+
+        return {
+            start: new Date(start),
+            end: new Date(end),
+        };
+    }, [deployedERUResponse]);
+
+    const dateRange = useMemo(() => {
+        if (
+            isNotDefined(deployedERUResponse)
+        || isNotDefined(deployedERUResponse.results)
+        || deployedERUResponse.results.length === 0
+        ) {
+            return undefined;
+        }
+
+        const startDateList = deployedERUResponse.results.flatMap(
+            ({ appeals }) => appeals?.map(
+                (appeal) => (isValidDate(appeal.start_date)
+                    ? new Date(appeal.start_date).getTime()
+                    : undefined),
+            ) ?? [],
+        ).filter(isDefined);
+
+        const endDateList = deployedERUResponse.results.flatMap(
+            ({ appeals }) => appeals?.map(
+                (appeal) => (isValidDate(appeal.end_date)
+                    ? new Date(appeal.end_date).getTime()
+                    : undefined),
+            ) ?? [],
+        ).filter(isDefined);
+
+        const start = minSafe([...startDateList, ...endDateList]);
+        const end = maxSafe([...startDateList, ...endDateList]);
+
+        if (isNotDefined(start) || isNotDefined(end)) {
+            return undefined;
+        }
+
+        return {
+            start: new Date(start),
+            end: new Date(end),
+        };
+    }, [deployedERUResponse]);
+
+    const handleExpandClick = useCallback(
+        (row: ERUDeploymentListItem) => {
+            setExpandedRow(
+                (prevValue) => (prevValue?.id === row.id ? undefined : row),
+            );
+        },
+        [],
+    );
+    const baseColumns = useMemo(() => ([
+        createLinkColumn<ERUDeploymentListItem, number>(
+            'name',
+            strings.deployedERUEmergency,
+            (item) => item.name,
+            (item) => ({
+                to: 'emergenciesLayout',
+                urlParams: {
+                    emergencyId: String(item.id),
+                },
+            }),
+        ),
+        createStringColumn<ERUDeploymentListItem, number>(
+            'organisation',
+            strings.deployedERUOrganisation,
+            () => '',
+            { columnClassName: styles.organisation },
+        ),
+        createMultiTimelineColumn<ERUDeploymentListItem, number>(
+            'timeline',
+            appealDateRange,
+            deployedEruDateRange ?? { start: new Date(), end: new Date() },
+            (item) => ({
+                startDate: item.appeals?.[0]?.start_date,
+                endDate: item.appeals?.[0]?.end_date,
```

Comment:
We need to iterate over all appeals and determine the minimum start date and the maximum end date.